modules/admin_manual/pages/installation/system_requirements.adoc

```diff
@@ -97,21 +79,13 @@ with a MySQL or MariaDB database:
 xref:configuration/database/linux_database_configuration.adoc#mysql-mariadb-storage-engine[MySQL / MariaDB storage engine])
 * `READ COMMITED` transaction isolation level (See: 
 xref:configuration/database/linux_database_configuration.adoc#mysql-mariadb-read-commited-transaction-isolation-level[MySQL / MariaDB `READ COMMITED` transaction isolation level])
-====
-
 
-=== Mobile
+== Memory Requirements
 
-* iOS 9.0+
-* Android 4.4+
```

Comment:
These seem to have disappeared. Is that correct? Are they somewhere else, in some other document?